apps/frontend/src/app/modules/canteen/reducer/canteen/canteen.reducer.ts

```diff
@@ -0,0 +1,73 @@
+import { CanteenAction, CanteenActionTypes } from './canteen.actions'
+import { getDay } from 'date-fns'
+
+export const CANTEEN_FEATURE_KEY = 'canteen'
+
+/**
```

Comment:
This whole comment should be removed when we first define the entity.

Reply:
Removed